src/modules/Calendar/Calendar.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useCallback, useMemo } from 'react';
+import Calendar from 'react-calendar';
+import { daysArr } from '../../mocks';
+import 'react-calendar/dist/Calendar.css';
+import './calendarStyles/calendar.css';
+
+export function CalendarStyled({ value, onChange }) {
+  // const [calendarValue, setCalendarValue] = useState(new Date());
+  const getShortWeekday = useCallback((locale, date) => daysArr[date.getDay()], []);
+  const minDate = useMemo(() => new Date(), []);
+
+  return (
+    <div className="calendar-container">
```

Comment:
why is not styled components?

Reply:
because this component from external library and these styles are used internally (I obviously don't see them in order to apply the styled-component)